scripts/exgfs_chem_init_aerosol.py

```diff
@@ -84,17 +87,55 @@ def get_env_var(varname: str, fail_on_missing: bool = True) -> str:
 def get_atm_files(path: str) -> typing.List[str]:
 	print(f'Checking for atm files in {path}')
 
-	files = list(map(lambda tile: atm_file_pattern.format(tile=tile, path=path), tiles))
-	for file_name in files:
+	file_list = []
+	for file_pattern in atm_file_pattern, atm_ctrl_pattern:
+		files = list(map(lambda tile: file_pattern.format(tile=tile, path=path), tiles))
+		for file_name in files:
+			if(debug):
+				print(f"\tChecking for {file_name}")
+			if(not os.path.isfile(file_name)):
+				print("FATAL: Atmosphere file {file_name} not found".format(file_name=file_name))
+				exit(101)
+			elif(debug):
+				print(f"\t\tFound {file_name}")
+		file_list = file_list + [ files ]
```

Comment:
No padding to separate square brackets from variable (PEP-8 standard)

Reply:
Done in 5a00777.